### ISSUE TITLE
[Feature request] Farm tick changer

### ISSUE THREAD

**Issue by linkdest** [Ephenia/Pokeclicker-Scripts#172]
The actual farm tick is 1.5s. It would be good to change that to make berries grow, mutate, get wanderer and farm coins faster or make it slow for the current bonus last more.

I don't know if it is possible change the tick without making a fork. But i saw that you were able to change the shiny chance constant. It would be really great if it is possible and you consider doing. Thanks for the current script.

**Comment by Ephenia** [comment]
Yeah, I'm sure that it's surely possible and I'll be considering this.

On a side note, I would expect to see more scripts like `catchfilterfantasia` or `syntheticshinysynapse` in the future.

The script `syntheticshinysynapse` in particular, is actually a great example of how things can be manipulated and how parts of the source can be modified, and also I suppose showing how creative that I would be with finding innovative ways to accomplish such a thing with what all of what its doing with minimal modification. A lot of my motivation for starting this project initially was trying to provide easily manageable modifications of the game that doesn't take needing to play a fork of the game that's been modified just to experience something different, and I wanted this project to serve as a good example for that. I also had been aware of forks of the game that quickly become outdated and not maintained well. Not that I blame maintainers, since keeping up with your own fork of the game with the main source updating a lot really isn't easy, it's a lot of work, and not too many understand that.

I just thinks scripts are the best and most manageable thing you can do when it comes to modifying a game like this, anyway. Not only is it all easily manageable with updating them alongside the game if anything breaks (rarely anything has too by the way), but they also allow for a much more and complete modular experience for the end user to customize their game.
